Makefile

```diff
@@ -54,7 +54,7 @@ help: ## Display this help.
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	$(CONTROLLER_GEN) crd:generateEmbeddedObjectMeta=true crd:maxDescLen=0 paths="./..." output:crd:artifacts:config=chart/templates/crds/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=chart/templates/crds/
```

Comment:
@itamar-marom with this CRD definition are you able to create pvc's ? by specifying metadata in volumeClaimTemplates ?

Reply:
I'll test it with Savhanna McLellan's Druid and will update you.